persim/bottleneck.py

```diff
@@ -101,7 +98,7 @@ def bottleneck(dgm1, dgm2, matching=False):
 
     # Step 2: Perform a binary search + Hopcroft Karp to find the
     # bottleneck distance
-    ds = np.sort(np.unique(D.flatten()))[0:-1] # Everything but np.inf
```

Comment:
@ctralie This is the only real change in this PR. The comment "Everything but np.inf" was confusing to me because of all the infinite handling that is done above. In particular, I don't think any np.inf's can make it to this point in the calculation. Wehn I comment out the `[0:-1]` slicing, all the tests still pass and the newly added test corresponding to the issue still pass. Thoughts on what this `[0:-1]` slice is doing here?

Reply:
Yes, you are absolutely right, great catch!  I put in some code above to catch infinite lifetimes to resolve a previous issue that was raised, but it looks like I forgot to get rid of the slice in question.  Luckily I had some comments from before :).  Thank you again

Reply:
Awesome, great!